src/Components/AllArticles.jsx

```diff
@@ -38,6 +39,28 @@ const AllArticles = () => {
     );
   };
 
+  const handleSortChange = (e) => {
+    const sortKey = e.target.value;
+    setSortBy(sortKey);
+    const sortedArticles = [...articles];
+
+    if (sortKey === "byDate") {
+      sortedArticles.sort(
```

Comment:
were doing all this functionality in the back end so we dont need to do any sorting this side :)